worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -70,6 +67,8 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
 
   val hasS3Storage = conf.hasS3Storage
 
+  val diskReserveSize = conf.workerDiskReserveSize
```

Comment:
Seems we should use `min(conf.workerDiskReserveSize, (totalSpace * diskReserveRatio.get` as `getMinimumUsableSize` does.

Reply:
done, use the `getMinimumUsableSize` method to replace the computation in `StorageManager`